workspaces/ui-v2/public/example-sessions/todos-partial.json

```diff
@@ -1785,6 +1785,94 @@
           "createdAt": "2020-10-20T20:52:31.794Z"
         }
       }
+    },
+    {
+      "BatchCommitStarted": {
```

Comment:
I'm not sure if there's a better way to generate commands (I just copy pasted from above and generated these by hand which seems to have worked)

Reply:
typically we like to dump it from the ui state, so it is transitively/inductively valid